### DISCUSSION TITLE
Boxed UncheckedBind

### DISCUSSION THREAD

**Discussion by SneakyBerry** [diesel-rs/diesel#3215] [Q&A] [answered]
Is it possible to make conditional raw query?
e.g
```rust
let mut query = sql("SELECT * FROM a WHERE a.a = ?").bind::<Integer, _>(10);
if some_cond {
    query = query.sql("AND b = ?").bind::<Integer, _>(20);
}
```

**Comment by weiznich** [comment, marked as answer]
Only using the upcoming 2.0 release. Checkout the 2.0.0-rc.0 version.
You can implement something like that by using [`SqlQuery::into_boxed()`](http://docs.diesel.rs/2.0.x/diesel/query_builder/struct.SqlQuery.html#method.into_boxed)
